### PR TITLE
Gsuite ia tsearch acordian iatid

### PR DESCRIPTION
**Pull request by Felixim0**

### CODE REVIEW

File: app/indexhandler.py [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 from app.searchcriteria import retrieve_job_roles, clear_stored_search_criteria
 
+from app.pageutils import (
```
Comment (SammyPoot) [inline app/indexhandler.py]: This can all be turned into one line.

File: app/interfaceactiontablehandler.py [hunk 2]
```diff
@@ -8,11 +8,16 @@
 from aiohttp_session import get_session
 from structlog import get_logger
 
+from app.pageutils import (
```
Comment (SammyPoot) [inline app/interfaceactiontablehandler.py]: Ditto, one line will do.

File: app/interfaceactiontablehandler.py [hunk 3]
```diff
@@ -102,14 +98,17 @@ async def get(self, request):
             employee_records = iat_employee_record_table(get_employee_info.json())
 
             job_role_json = retrieve_job_roles(get_job_roles, '')
-
+            
+            iatStat = retreiveIATstatuses()
```
Comment (SammyPoot) [inline app/interfaceactiontablehandler.py]: This method isn't named consistently. It should probably be `retreive_iat_statuses`.

Comment (SammyPoot) [reply app/interfaceactiontablehandler.py]: The variable name should probably also be `iat_stats`

File: app/interfaceactiontablehandler.py [hunk 4]
```diff
@@ -229,6 +232,8 @@ async def post(self, request):
             job_role_short_json = retrieve_job_roles(
                 get_job_roles, previous_jobrole_selected)
 
+            iatStat = retreiveIATstatuses()
```
Comment (SammyPoot) [inline app/interfaceactiontablehandler.py]: Ditto, could use renaming.

File: app/interfaceactiontablehandler.py [hunk 5]
```diff
@@ -245,7 +250,8 @@ async def post(self, request):
                 'previous_badge': previous_badge,
                 'previous_jobid': previous_jobid,
                 'previous_surname_filter': previous_surname,
-                'no_employee_data': no_employee_data
+                'no_employee_data': no_employee_data,
+                'iat_options': iatStat,
```
Comment (SammyPoot) [inline app/interfaceactiontablehandler.py]: Ditto, could use renaming.

File: app/interfaceactiontablehandler.py [hunk 6]
```diff
@@ -351,6 +357,7 @@ async def get(self, request):
 
             job_role_json = retrieve_job_roles(get_job_roles,
                                                previous_jobrole_selected)
+            iatStat = retreiveIATstatuses()
```
Comment (SammyPoot) [inline app/interfaceactiontablehandler.py]: Ditto, could use renaming.

File: app/interfaceactiontablehandler.py [hunk 7]
```diff
@@ -367,7 +374,8 @@ async def get(self, request):
                 'previous_firstname': previous_firstname,
                 'previous_badge': previous_badge,
                 'previous_jobid': previous_jobid,
-                'previous_surname_filter': previous_surname
+                'previous_surname_filter': previous_surname,
+                'iat_options':iatStat,
```
Comment (SammyPoot) [inline app/interfaceactiontablehandler.py]: Ditto, could use renaming.

File: app/searchcriteria.py [hunk 8]
```diff
@@ -84,6 +82,19 @@ def retrieve_job_roles(job_roles, previous_jobrole_selected):
     return add_job_roles
 
 
+def retreiveIATstatuses():
```
Comment (SammyPoot) [inline app/searchcriteria.py]: Ditto, could use renaming.

Comment (SammyPoot) [reply app/searchcriteria.py]: If this only returns static data, should this be a function?

File: app/searchfunctions.py [hunk 9]
```diff
@@ -37,9 +43,9 @@ def get_employee_records_no_device(user_filter=""):
                         verify=False,
                         auth=HTTPBasicAuth(FSDR_USER, FSDR_PASS))
 
-def get_employee_records(user_filter="", calledFromIAT=False):
+def get_employee_records(user_filter="", iat=False):
     employee_record_url = URL(
-        FSDR_URL + f'/fieldforce/byType/byRangeAndUserFilter' + str("Iat/" if calledFromIAT else "/") ).with_query(
+        FSDR_URL + f'/fieldforce/byType/byRangeAndUserFilter' + str("Iat/" if iat else "/") ).with_query(
```
Comment (SammyPoot) [inline app/searchfunctions.py]: F-strings can contain code directly: `f"{1 if True else 2}"`
You can replace this with `f'/fieldforce/byType/byRangeAndUserFilter{"Iat/" if iat else "/"}'`

**Review by SammyPoot — CHANGES_REQUESTED** [review]

File: app/searchfunctions.py (hunk 9 above)
Comment (Felixim0) [reply app/searchfunctions.py]: Done

File: app/interfaceactiontablehandler.py (hunk 3 above)
Comment (Felixim0) [reply app/interfaceactiontablehandler.py]: Changed to iat_stats = retreive_iat_statuses()

File: app/interfaceactiontablehandler.py (hunk 7 above)
Comment (Felixim0) [reply app/interfaceactiontablehandler.py]: Done

File: app/searchcriteria.py (hunk 8 above)
Comment (Felixim0) [reply app/searchcriteria.py]: Perhaps. Originally it was to be  dynamically allocated and get all distinct values from the database, however I realised then you wouldn't be  able to see all the options if someone wasn't assigned it. It  might also be unneeded stress on the database.

**Review by SammyPoot — APPROVED** [review]

**Pull request merged**